docs/content/doc/packages/cargo.en-us.md

```diff
@@ -0,0 +1,109 @@
+---
+date: "2022-11-20T00:00:00+00:00"
+title: "Cargo Packages Repository"
```

Comment:
I used this name because other package software like Artifactory prefer Cargo too:
https://www.jfrog.com/confluence/display/JFROG/Cargo+Package+Registry
https://cloudsmith.com/blog/native-cargo-uploads-in-cloudsmith/
https://github.com/sonatype-nexus-community/nexus-repository-cargo
https://hirevo.github.io/alexandrie/introduction.html